src/test/java/org/synyx/urlaubsverwaltung/core/sync/providers/exchange/ExchangeCalendarProviderTest.java

```diff
@@ -1,12 +1,28 @@
 package org.synyx.urlaubsverwaltung.core.sync.providers.exchange;
 
+import microsoft.exchange.webservices.data.core.ExchangeService;
+import microsoft.exchange.webservices.data.core.enumeration.misc.ExchangeVersion;
+import microsoft.exchange.webservices.data.core.enumeration.property.WellKnownFolderName;
+import microsoft.exchange.webservices.data.core.service.folder.CalendarFolder;
+import microsoft.exchange.webservices.data.core.service.item.Appointment;
+import microsoft.exchange.webservices.data.property.complex.Attendee;
+import microsoft.exchange.webservices.data.property.complex.AttendeeCollection;
+import microsoft.exchange.webservices.data.property.complex.FolderId;
+import microsoft.exchange.webservices.data.property.complex.ItemId;
+import microsoft.exchange.webservices.data.search.FindFoldersResults;
 import org.junit.Test;
 import org.mockito.Mockito;
 import org.synyx.urlaubsverwaltung.core.mail.MailService;
+import org.synyx.urlaubsverwaltung.core.person.Person;
 import org.synyx.urlaubsverwaltung.core.settings.CalendarSettings;
 import org.synyx.urlaubsverwaltung.core.settings.ExchangeCalendarSettings;
+import org.synyx.urlaubsverwaltung.core.sync.absence.Absence;
 
-import static org.mockito.Mockito.when;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.*;
```

Comment:
remove star import. Maybe use for all Mockito.* static imports

Reply:
done